test-files/cases/differential/unifying/highway/roundabout-3580/Config.conf

```diff
@@ -1,4 +1,5 @@
 {
```

Comment:
Decided to leave the whole match removal enabled for this one...kind of cheating. Its kind of tough to tell if the output got better or worse w/o this config change since the two inputs are so different in spots.